packages/edit-site/src/components/sidebar/global-styles/style.scss

```diff
@@ -15,3 +15,19 @@
 	}
 }
 
+.edit-site-global-style-palette {
+	margin: $grid-unit-20;
+
+	.component-color-indicator {
```

Comment:
We keep duplicating these styles for the color indicator component, it seems it needs a `isRound` and a `size` prop maybe. Potentially make all indicators round.

Reply:
All color indicators are circular now, no?
cc @ciampo

Reply:
That component is an old one that is used for the old color panels (when closed they show a rectangle with a color), it's the only component that is exposed AFAIK. It feels we should just update it to be circular.

Reply:
That's my inclination as well

Reply:
I've been taking a look at these inconsistencies and discussed them with a few folks (including @mtias) — I reported the outcome of that conversation in https://github.com/WordPress/gutenberg/issues/33820#issuecomment-904497769.

The TL;DR was that we agreed to look into creating a new component, called `ColorSwatch`, that would potentially replace both `ColorIndicator` and `CircularOptionPicker`, offering a better API and better a11y. It will be then used as the base for rewriting/improving higher-level components for managing color palettes.

I'm currently in the process of creating a tracking issue #35093 for color-related components which will includes the tasks related to `ColorSwatch` & co.

Reply:
Thanks for the refresher — if it's all tracked we are good.